app/models/peoplefinder/concerns/completion.rb

```diff
@@ -21,7 +15,7 @@ def completion_score_fields
       lambda {
         where(%{
           COALESCE(image,'') = ''
-          OR COALESCE(location,'') = ''
+          OR COALESCE(location_in_building,'') = ''
```

Comment:
I don't know whether this is appropriate for determining "inadequate profiles": I'm not sure how this is used.